htdocs/install/mysql/migration/15.0.0-16.0.0.sql

```diff
@@ -30,6 +30,10 @@
 -- -- VPGSQL8.2 SELECT dol_util_rebuild_sequences();
 
 
+
+ALTER TABLE llx_holiday ADD COLUMN nb_open_day double(24,8)     DEFAULT 0;
```

Comment:
According to announcement done during last devcamp, change into the structure of tables must be pushed into a different PR, before the PR that change the PHP code.